src/feed/feed.controller.ts

```
@@ -69,4 +73,15 @@ export class FeedController {
   remove(@Param('id') id: string) {
     return this.feedService.remove(+id);
   }
+
+  @Get('reports')
+  @ApiBearerAuth('jwt-auth')
+  @UseGuards(JwtAuthGuard, RolesGuard)
+  @RoleDecorator(UserRole.ADMIN)
+  @ApiOperation({ summary: 'Get paginated reported content for admin review' })
+  @ApiResponse({ status: HttpStatus.OK, description: 'Returns paginated reported content' })
```

ESLint

[error] Replace `·status:·HttpStatus.OK,·description:·'Returns·paginated·reported·content'` with `⏎····status:·HttpStatus.OK,⏎····description:·'Returns·paginated·reported·content',⏎·`

(prettier/prettier)